app/views/LocationTracking.js

```diff
@@ -125,11 +125,13 @@ class LocationTracking extends Component {
       });
   }
 
-  checkIfUserAtRisk() {
+  spawnIntersect() {
```

Comment:
Splitted the function into two parts, `spawnIntersect()` is the heavyweight intersection algo call while `checkIfUserAtRisk()` only reads from asyncstorage and changes the results based on the state.